crypto/evp_extra/internal.h

```diff
@@ -22,13 +22,11 @@ typedef struct {
   char has_private;
 } X25519_KEY;
 
-#define KYBER512_PUBLICKEY_BYTES 800
-#define KYBER512_SECRETKEY_BYTES 1632
 typedef struct {
-  uint8_t pub[KYBER512_PUBLICKEY_BYTES];
-  uint8_t priv[KYBER512_SECRETKEY_BYTES];
+  uint8_t pub[800];
```

Comment:
magic number

Reply:
I normally prefer not to have it like this either, but I deferred to the convention of this file (see the other key structures above this).